### ISSUE TITLE
Allow me to specify the database to save LogEntry

### ISSUE THREAD

**Issue by SChiang-VA** [jazzband/django-auditlog#272]
Due to the volume of audit logs and integration with legacy database, I need my audit logs in a separate database.

I suggest adding a LOGGING_DATABASE_NAME variable and amending the LogEntryManager such as:
`             # save LogEntry to same database instance is using unless settings.LOGGING_DATABASE_NAME is specified
            db = getattr(settings, 'LOGGING_DATABASE_NAME', instance._state.db)
            return self.create(**kwargs) if db is None or db == '' else self.using(db).create(**kwargs)`

instead of:
`            # save LogEntry to same database instance is using
            db = instance._state.db
            return self.create(**kwargs) if db is None or db == '' else self.using(db).create(**kwargs)`

**Comment by ramanjaneyalu** [comment]
Will this work, because tables which are present in other db are not being entered in the auditlog_logentry and django_content_type tables. If it is not present then we will not be able to store these entries as django_content_type has no entry of that table.

Correct me if i am wrong.

**Comment by SChiang-VA** [comment]
In my scenario, my operational data resides in a legacy database while I'd need to store my auditlog in my default db because I'm not allowed to create new tables in the legacy database. Hence, I was able to solve my problem when I set LOGGING_DATABASE_NAME to default in my setting.py after I made my suggested changes.

But I understand where you are coming from. As the LogEntry model has 2 foreignkeys, if I were to specify LOGGING_DATABASE_NAME to be a database without the ContentType and AUTH_USER_MODEL tables it will not work.

Perhaps, then the assumption to "save LogEntry to same database instance is using" is not correct to begin with?

**Comment by ramanjaneyalu** [comment]
Yes, currently i am facing the issue where i have 2 databases but audit_logging is present in default database and i am unable to migrate to the second database. Due to which it is not allowing me to save the data as well. Is there a workaround for this scenario.

Could you please suggest is there a way to migrate the 2 tables to the second db, so that audit logging might be possible without errors.

**Comment by kbiernat** [comment]
+1
I have a local sqlite db (default) and an existing db, added models reflecting it, with Managed=False setting.
It looks like auditlog is trying to write into the external DB instead of sqlite.

1142, "INSERT command denied to user 'someuser'@'10.x.x.x' for table 'auditlog_logentry'"
The auditlog table exists in sqlite, so this part is correct.
